source/Halibut.Tests/HalibutTimeoutsAndLimitsForTestsBuilder.cs

```diff
@@ -49,6 +49,8 @@ public HalibutTimeoutsAndLimits Build()
                 limits.MaximumActiveTcpConnectionsPerPollingSubscription = maximumActiveTcpConnectionsPerPollingSubscription.Value;
             }
 
+            limits.UseAsyncListener = true;
```

Comment:
This feature is optional, but we are turning it on by default for our tests.

I think it makes more sense to run our tests the way they will be used by Octopus (i.e., we this off).

In saying that, it would be nice to see all the tests work with this on as well. Would it be overkill to double the number of tests and create different test cases with this on and off? 🤔 

Or do we just make a few choice tests with this on for now?

Reply:
I think the direction would be to have this on so if anything we would want this on, not off.

Do we want a one off run with it off to check all that behaviour remains?

Reply:
If the plan is to move it towards being the default behaviour, then this is fine.